examples/mm/stable_diffusion/conf/draftp_sd.yaml

```diff
@@ -0,0 +1,240 @@
+name: sd_draftp_train
+
+trainer:
+  devices: 1
+  num_nodes: 1
+  accelerator: gpu
+  precision: bf16
+  logger: False # logger provided by exp_manager
+  enable_checkpointing: False
+  # replace_sampler_ddp: False
+  max_epochs: 50000000 # PTL default. In practice, max_steps will be reached first.
+  max_steps: 50 # consumed_samples = global_step * micro_batch_size * data_parallel_size * accumulate_grad_batches
+  log_every_n_steps: 1
+  accumulate_grad_batches: 1 # do not modify, grad acc is automatic for training megatron models
+  gradient_clip_val: 1.0
+  benchmark: False
+  enable_model_summary: True
+  val_check_interval: 20
+
+exp_manager:
+  exp_dir: null
+  name: ${name}
+  explicit_log_dir: null
+  create_wandb_logger: False
+  wandb_logger_kwargs:
+    project: NeMo-draft
+    #group: nemo-sd
+    name: ${name}
+    resume: False
+  create_checkpoint_callback: True
+  create_tensorboard_logger: True
+  checkpoint_callback_params:
+    every_n_train_steps: 50
+    every_n_epochs: 0
+    monitor: reduced_train_loss
+    filename: '${name}--{reduced_train_loss:.2f}-{step}-{consumed_samples}'
+  resume_if_exists: True
+  resume_ignore_no_checkpoint: True
+  ema:
+    enable: True
+    decay: 0.9999
+    validate_original_weights: False
+    every_n_steps: 1
+    cpu_offload: False
+
+reward_model: "pickscore"
+
+pretrained_checkpoint:
+  restore_from_path: null
+
+model:
+  val_check_interval: 0
+  limit_val_batches: 0
+  save_interval: 500
+  kl_coeff: 10
+  truncation_steps: 1
+  gradient_clip_val: 1000000
+  max_epochs: 5 # PTL default. In practice, max_steps will be reached first.
+  max_steps: 1000000
```

Comment:
it's better if they follow the aligner convention and be put under trainer.draft_sd.<<<ARGS>>>